impl/storage.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package impl
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/spanner"
+	"github.com/google/keytransparency/core/sequencer"
+	"github.com/google/keytransparency/core/water"
+	"github.com/google/keytransparency/impl/mysql/mutationstorage"
+	"github.com/google/keytransparency/impl/spanner/batch"
+	"github.com/google/keytransparency/impl/spanner/directory"
+	"github.com/google/keytransparency/impl/spanner/mutations"
```

Comment:
Can we import these with custom names? In the code below it should be clearer which are mysql and which are spanner. `batch` doesn't help me.

Reply:
Good idea. done